src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Ball {
+    private final Colors color;
+    private final int number;
+
+    public Ball(Colors color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public String getBallInfo() {
+        return new StringBuilder()
+                .append("Ball color = ")
+                .append(color)
+                .append(", ")
+                .append("number = ")
+                .append(number)
+                .toString();
+    }
```

Comment:
it would be better to override `toString` method instead of creating this one (because `toString` is a standard method for string representation of an object)

also, it is better to use `color.name()` 
(`.append(color)` will call `toString`⬇️ )
[Use name() for getting String representation of enum constants](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants)